packages/perseus/src/widgets/table/table.tsx

```diff
@@ -18,6 +18,7 @@ import type {
     PerseusTableRubric,
     PerseusTableUserInput,
 } from "@khanacademy/perseus-score";
+import {getTablePublicWidgetOptions} from "@khanacademy/perseus-core";
```

Comment:
Might want to merge this import with the other `perseus-core` ones. My IDE gives me warnings about this sometimes, but not always. I don't know if I checked this for your other PRs, so those might have two as well.

Reply:
It looks like the only other import from `@khanacademy/perseus-core` is types-only. I think our linter requires type and value imports to be separated like this? The alternative would be:

```ts
import {
    getTablePublicWidgetOptions,
    type PerseusTableWidgetOptions,
} from "@khanacademy/perseus-core";
```

Reply:
I usually write it the alternative way you show here. If both are correct though, then totally okay with either. I felt it would be good to keep imports from the same place together if possible, so I combine them when I notice.

Reply:
I think when the linter auto fixes/auto imports, it splits them... but the linter doesn't force that separation.  I tend to prefer type imports separated, but that's personal preference. :)